src/metadata/misc_parsers.cpp

```diff
@@ -180,7 +180,7 @@ void LibRaw::parse_cine()
   }
   cam_mul[0] = getreal(LIBRAW_EXIFTAG_TYPE_FLOAT);
   cam_mul[2] = getreal(LIBRAW_EXIFTAG_TYPE_FLOAT);
-  maximum = ~((~0u) << get4());
+  maximum = ~((~0u) << MIN(get4(), 31));
```

Comment:
Finally found [+] button in diff.

Why not LIM(get4(),0,31)?

Reply:
Good idea, I noticed in #126ce46 you used a lower bound of 1 and not 0, is there a reason for this?

Reply:
real bit depth is always in 8..16 bit range, so both 0 and 1 are equally (almost) OK.